crates/nu-command/tests/commands/platform/du.rs

```diff
@@ -0,0 +1,23 @@
+use nu_test_support::nu;
+
+#[test]
+fn test_du_flag_min_size() {
+    let actual = nu!("du -m -1");
+    assert!(actual
+        .err
+        .contains("Negative value passed when positive one is required"));
+
+    let actual = nu!("du -m 1");
+    assert!(actual.err.is_empty());
+}
+
+#[test]
+fn test_du_flag_max_depth() {
+    let actual = nu!("du -d -2");
+    assert!(actual
+        .err
+        .contains("Negative value passed when positive one is required"));
+
+    let actual = nu!("du -d 2");
+    assert!(actual.err.is_empty());
```

Comment:
Thanks for the tests. Do we need to constrain what files `du` will try to look at? It shouldn't traverse half the file system for a small unit test of flags.

Reply:
How can we do that ? 
Shall we remove successfull result tests ?

Reply:
Either that or pointing the `cwd` to a directory with not much in it.

Reply:
Got it 
Will resolve